crypto/evp/c_all.c

```diff
@@ -82,9 +82,4 @@ void OPENSSL_add_all_algorithms_noconf(void)
     OPENSSL_cpuid_setup();
     OpenSSL_add_all_ciphers();
     OpenSSL_add_all_digests();
-#ifndef OPENSSL_NO_ENGINE
-# if defined(__OpenBSD__) || defined(__FreeBSD__) || defined(HAVE_CRYPTODEV)
-    ENGINE_setup_bsd_cryptodev();
-# endif
-#endif
```

Comment:
Thanks for review. I'll update the pull request and look for a port on master.

On a side note - I keep seeing non-related build errors. Should I do something about this - like rebase and hope for a better behaved build?